frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -440,7 +440,7 @@ function LivenessProbes(props: { liveness: KubeContainer['livenessProbe'] }) {
   return (
     <Box display="flex" flexDirection="column">
       <LivenessProbeItem>
-        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port}, 
+        {`http-get, path: ${liveness?.httpGet?.path}, port: ${liveness?.httpGet?.port},
```

Comment:
I think this should not belong here

Reply:
Yeah. Our lint checks should have detected this whitespace before we pushed it to main, but I will add a new commit just fixing that.